python/docs/source/examples/time_centric.ipynb

```diff
@@ -81,7 +81,7 @@
     "## Kaskada Client Setup\n",
     "\n",
     "```\n",
-    "%pip install kaskada>=0.6.0-a.1\n",
+    "%pip install kaskada>=0.6.0-a.2\n",
```

Comment:
We shouldn't need to explicitly do this unless we want to -- I think `>=` will take the latest pre-release, which would be `0.6.0-a.2` once it's available.